src/main/java/com/io/image/manager/controller/ImageController.java

```diff
@@ -22,15 +22,22 @@
 
 import javax.servlet.http.HttpServletRequest;
 import java.awt.image.BufferedImage;
+import java.io.BufferedWriter;
+import java.io.ByteArrayOutputStream;
+import java.io.FileWriter;
 import java.io.IOException;
+import java.sql.Timestamp;
 import java.util.List;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 @RestController
 public class ImageController {
     private final ImageService imageService;
     private final DistributionSummary outboundTrafficSummary;
     private final AppConfigurationProperties props;
+    private final String logPath = "/log/IM_log.txt";
```

Comment:
Move that path to application config and access it from `AppConfigurationProperies` (the `props` field above)